infer/SigSuggestion.cc

```diff
@@ -328,9 +328,7 @@ bool childNeedsOverride(core::Context ctx, core::SymbolRef childSymbol, core::Sy
         //  that is either overridable...
         (parentSymbol.data(ctx)->isOverridable() ||
          // or override...
-         parentSymbol.data(ctx)->isOverride() ||
-         // or implementation.
-         parentSymbol.data(ctx)->isImplementation());
+         parentSymbol.data(ctx)->isOverride());
```

Comment:
We could clean up a bunch of this code if we *also* got rid of `overridable` but that's out of scope.